### DISCUSSION TITLE
Colour Change Bug

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#16] [Fixed Issues]
Am I the only user having this problem?

If I click on the colour change icon next to playback settings, it brings up the Webradio library page plus the colour selector. If I am not _very_ quick to click on the colour selector, rAudio will do a replace and play for the first webradio entry on the library list.

Previously, the colour selector appeared over an album track list and worked well.

**Comment by AndyFx12** [comment]
Same here. 
regards, Andy

**Comment by rern** [comment]
Fix with today's update.

**Reply by R101G** [reply]
Thanks rern.
